hot-reload-orchestration/src/main/kotlin/org/jetbrains/compose/reload/orchestration/OrchestrationMessage.kt

```diff
@@ -213,14 +213,25 @@ public sealed class OrchestrationMessage : Serializable {
 
     public data class ReloadClassesResult(
         val reloadRequestId: UUID,
-        val isSuccess: Boolean,
+        val type: ResultType,
         val errorMessage: String? = null,
         val errorStacktrace: List<StackTraceElement>? = null,
     ) : OrchestrationMessage() {
+        public val isSuccess: Boolean get() = type == ResultType.Success
+
         internal companion object {
             @Suppress("unused")
             internal const val serialVersionUID: Long = 0L
         }
+
+        public enum class ResultType {
+            Success, VerificationError, Failure;
+
+            internal companion object {
+                @Suppress("unused")
+                internal const val serialVersionUID: Long = 0L
+            }
+        }
```

Comment:
So far, I think the only usage of this is in the devtools to render a different message, right? Are you sure you want to change (and commit) to this breaking change to the orchestration? 

Alternatively, the errorMessage carried here could just be groomed before sending the message!

Reply:
Yes, this was also one of my concerns. Thats why I wanted to hear your opinion on it. I feel like adding more explicit result type can be (potentially) benefitial, but we don't have any real use cases for it right now